README.md

```diff
@@ -231,6 +232,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [LoanDolphin](https://loandolphin.com.au/) [:rocket:](https://loandolphin.breezy.hr/) | Home loan marketplace. | `Lisboa` `Remote` |
 | [Natixis](https://www.natixis.com/) [:rocket:](https://www.natixis.com/natixis/jcms/tki_5051/en/careers) | French corporate and investment bank. | `Porto` |
 | [Onfido](https://www.onfido.com) [:rocket:](https://onfido.com/jobs/) | Identity verification at the speed of life. | `Lisboa` |
+| [Revolut](https://www.revolut.com) [:rocket:](https://www.revolut.com/pt-PT/careers) | A digital banking alternative to conventional banks. | `Porto` |
```

Comment:
If I am not mistaken Revolut's offices in Porto are not towards Software Development, right?

Reply:
Does it matter? It’s still a tech company...

Reply:
If I'm not mistaken, it does. 
@marmelo can you deny or confirm?

Reply:
For context, #92 added a location (Lisboa) which had only a sales positions while adding other locations with "tech jobs".

I always understood this repo to list "tech companies" which have "tech jobs" in Portugal. Not to list "tech companies" which have an office in Portugal.

Some clarification and guidance on CONTRIBUTING.md would be appreciated.

Reply:
Exactly @vpereira01, I remembered that discussion on that PR, but after I took a look at CONTRIBUTING.md I found nothing to show it.

Reply:
Revolut has started hiring in Portugal for tech positions. So this is now OK to add to the list.